google-cloud-pub-sub/src/main/scala/akka/stream/alpakka/googlecloud/pubsub/HttpApi.scala

```diff
@@ -102,7 +102,7 @@ private trait HttpApi {
   ): Future[immutable.Seq[String]] = {
     import materializer.executionContext
 
-    val url: Uri = s"$PubSubGoogleApisHost/v1/projects/$project/topics/$topic:publish?key=$apiKey"
+    val url: Uri = s"$PubSubGoogleApisHost/v1/projects/$project/topics/$topic:publish"
```

Comment:
Key param seems to make auth fail - whereas, with it removed auth works nicely.
Regarding tests: there is a test defined in google-cloud-pub-sub/src/test/scala/akka/stream/alpakka/googlecloud/pubsub/HttpApiSpec.scala with mocking, that I updated as part of this PR. The only means to improve this to ensure the code 'really works' (that I can think of), is to actually hit the Google API, with real credentials - this could guard against any future behaviour changes. What do you think?